python_client/test.py

```diff
@@ -0,0 +1,56 @@
+
+# make install
+
+# currently CMAKE_INSTALL_LIBDIR=/home/ENV{USERNAME}/chronolog/lib
+
+## create symlink for  py_chronolog_client.[linux_distibution_version].so 
+## ln -s ${CMAKE_INSTALL_LIBDIR}/py_chronolog_client.so ${CMAKE_INSTALL_LIBDIR}/py_chronolog_client.[linux_distribution_version].so
+
+## update LD_LIBRARY_PATH
+## export LD_LIBRARY_PATH=${CMAKE_INSTALL_LIBDIR}:${LD_LIBRARY_PATH}
+## update PYTHONPATH
+## export PYTHONPATH=${CMAKE_INSTALL_LIBDIR}:${PYTHONPATH}
+
+
+## using spack installed python3 version
```

Comment:
pls consider being explicit to say using `${CHRONOLOG_SRC_DIR}/.spack-env/view/bin/python3`. as you mentioned, the system's python may fail due to mismatched version.